vm/systemSmartContracts/staking.go

```diff
@@ -88,16 +113,206 @@ func (r *stakingSC) Execute(args *vmcommon.ContractCallInput) vmcommon.ReturnCod
 		return r.jail(args)
 	case "unJail":
 		return r.unJail(args)
+	case "changeRewardAddress":
+		return r.changeRewardAddress(args)
+	case "changeValidatorKeys":
+		return r.changeValidatorKey(args)
 	}
 
 	return vmcommon.UserError
 }
 
+func getPercentageOfValue(value *big.Int, percentage float64) *big.Int {
```

Comment:
Do we really need this in this exact form? I would avoid percentage as float64. Can we have percentage as int and all operations done on ints? The main reason is that this code is treated as a "VM" code.

Reply:
we have some percentages here and there in the code as float64. We have to make a denomination if we go with INT. Open for change, but has to be done on the whole protocol.